data/transactions/logic/assembler.go

```diff
@@ -31,6 +31,7 @@ import (
 	"sort"
 	"strconv"
 	"strings"
```

Comment:
@jannotti Minor:  Are macros worthy of a spec modification (https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/README.md)?

I'm imagining something in the neighborhood of https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/README.md#constants being extended.

Reply:
Yes, I suppose so.  It's not _really_ a spec change (of the protocol) but since that's where we document some assembler level stuff, it would be the right place to explain these as well.  I'll add.

Reply:
I think we can merge without a spec mod, as they don't actually change the protocol.